packages/teams-js/src/public/tasks.ts

```diff
@@ -36,12 +41,15 @@ export namespace tasks {
         (sdkResponse: dialog.ISdkResponse) => submitHandler(sdkResponse.err, sdkResponse.result)
       : undefined;
     if (taskInfo.card !== undefined || taskInfo.url === undefined) {
```

Comment:
Now that we're moving all of these calls to dialog, we should make sure and be extra careful to not miss any cases. I think that before this change, if someone didn't specify card OR url, we would fall into line 39 and call sendMessageToParent which would let the host decide what to do with the incorrect set of parameters. After this change, we would be calling the dialog.adaptiveCard.open or dialog.adaptiveCard.bot.open functions, which I don't think will throw errors in the same way. We want to make sure that this function continues to fail the same way in that case.